src/components/navigation.tsx

```diff
@@ -7,76 +8,166 @@ import {
+                      <NavigationMenuItem>
+                        {subItems ? (
+                          <>
+                            <div className="flex w-full flex-col">
+                              <div className="flex w-full items-start justify-between">
+                                <Link
+                                  href={link}
+                                  className="text-base font-light"
+                                  onClick={handleNavBar}
+                                >
+                                  {name}
+                                </Link>
+                              </div>
+                              <div
+                                className={`${programBar ? "flex flex-col pt-3" : "hidden"}`}
+                              >
+                                {subItems.map(
+                                  ({ subName, subLink }, subIndex) => (
+                                    <Link
+                                      key={subIndex}
+                                      href={subLink}
+                                      className="py-1 text-sm font-light capitalize"
+                                      onClick={handleBothBars}
+                                    >
+                                      {subName.toLowerCase()}
+                                    </Link>
+                                  ),
+                                )}
+                              </div>
+                            </div>
+                          </>
+                        ) : (
+                          <Link
+                            href={link}
+                            className="text-base font-light"
+                            onClick={handleNavBar}
+                          >
+                            {name}
+                          </Link>
+                        )}
+                      </NavigationMenuItem>
+                    </NavigationMenuList>
+                  </NavigationMenu>
+                  {subItems && (
+                    <div className="text-acm-gray-100 absolute right-12.5 flex w-1/8 items-center justify-center border-l-1">
+                      {programBar ? (
+                        <RiArrowUpSLine
+                          className="text-acm-gray-400"
+                          size={24}
+                          onClick={handleProgramBar}
+                        />
+                      ) : (
+                        <RiArrowDownSLine
+                          className="text-acm-gray-400"
+                          size={24}
+                          onClick={handleProgramBar}
+                        />
+                      )}
+                    </div>
+                  )}
+                </div>
+              ))}
+            </div>
+          </div>
+          <div
+            className="h-full w-1/8 bg-white opacity-45"
+            onClick={handleNavBar}
+          ></div>
```

Comment:
good shit here. use a self closing tag though
```suggestion
          <div
            className="h-full w-1/8 bg-white opacity-45"
            onClick={handleNavBar}
          ></div>
```